README.rst

```diff
@@ -319,10 +320,32 @@ Admin mixins
   This is an admin mixin for models inheriting ``TimeStampedEditableModel``
   which adds the fields ``created`` and ``modified`` to the database.
 
+Test utilities
+^^^^^^^^^^^^^^
+
+This package contains some utilities that are used in the automated builds
+of different OpenWISP modules.
+
+``catch_signal``
+~~~~~~~~~~~~~~~~ 
+
+This method can be used to mock a signal call inorder to easily verify
+that the signal has been called.
+
+Usage example as a context-manager::
+
+    with catch_signal(status_signal) as handler:
+            <Perform the action that generates the signal>
+        handler.assert_called_once_with(
+            <parameters>=<arguments>,
+            sender=<Sender name>,
+            signal=<Signal name>,
+        )
```

Comment:
Please define code block language as done in [here](https://github.com/openwisp/openwisp-utils/pull/88/files#diff-88b99bb28683bd5b7e3a204826ead112R309). 

Making example copy paste friendly may help future contributors who are working with this feature for first time. What do you say @nepython ?
Also comment out helper strings as
```
<Perform the action that generates the signal>
``

Reply:
>Making example copy paste friendly may help future contributors who are working with this feature for first time. What do you say @nepython ?

Actually @TheOneAboveAllTitan, I have copy pasted the actual usage of this feature just replacing **specific names of signals and parameters** with help-text in **<>** as I found it as a general practice in openwisp and elsewhere. I am new to documenting changes, open to all suggestions to make it as user-friendly as possible:smiley:. Thanks for the suggestions.

Reply:
Sorry, I made a typo in the previous comment. :sweat: 
But you got it right anyway :+1: :smile: